pkg/synopsysctl/cmd_destroy.go

```diff
@@ -153,18 +166,8 @@ func destroy(namespace string) error {
 		}
 	}
 
-	// delete role bindings
-	for _, roleBinding := range roleBindings {
-		log.Infof("deleting role binding '%s'", roleBinding)
-		err = util.DeleteRoleBinding(kubeClient, namespace, roleBinding)
-
-		if err != nil {
-			log.Errorf("unable to delete role binding '%s' due to %+v", roleBinding, err)
-		}
-	}
-
 	// delete cluster roles
-	clusterRoles, roles, err := util.GetOperatorRoles(kubeClient, namespace)
+	clusterRoles, _, err := util.GetOperatorRoles(kubeClient, namespace)
```

Comment:
Delete cluster roles before role bindings.  You are using CRB in the RB delete logic, so want to use untouched CRB.

Reply:
as per the discussion, ignoring